services/app-api/utils/authorization.ts

```diff
@@ -0,0 +1,105 @@
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+import jwt_decode from "jwt-decode";
+// types
```

Comment:
did we want to remove `// types` here too?

Reply:
removed